### ISSUE TITLE
`bin/gemstash` fails with `Could not find command "readme"`

### ISSUE THREAD

**Issue by technicalpickles** [rubygems/gemstash#396]
This is specifically about running gemstash in development, but I'm able to do things like `bin/gemstash start`, but `bin/gemstash` or anything that would do help usage fails like:

```
❯ bin/gemstash                                                                                                                                                                                                  
Could not find command "readme".
```

This is the only place I can find readme mentioned:

https://github.com/rubygems/gemstash/blob/e0e1b4b920ac29b9f777e0cd0b9325e3d73f5259/lib/gemstash/cli.rb#L40-L49

**Comment by olleolleolle** [comment]
Perhaps the casing changed, and we need to support the uppercase README (md) file, here?

**Comment by technicalpickles** [comment]
Forgot to post this, but I was able to confirm that running `rake doc` does create the files it's looking for, ie `/Users/josh.nichols/workspace/gemstash/lib/gemstash/man/gemstash-readme.7`

This is definitely specific to local development. I'm not sure what, if anything, should happen to read from the local files in this case 🤔

**Comment by olleolleolle** [comment]
@technicalpickles Yeah, but it _is_ a weird location to use, isn't it? `lib/gemstash/man/` rather than `man/`, I mean.

**Comment by olleolleolle** [comment]
@technicalpickles Thanks, coming back to this, perhaps the message could explain "if you're in local dev, please regenerate docs"?

**Comment by technicalpickles** [comment]
Yeah, that makes sense to me 👍🏻  I was trying to think how you might determine that, and I think checking `$0 == "bin/gemstash"` might be the easiest way?
